.github/workflows/test.yaml

```diff
@@ -54,15 +54,15 @@ jobs:
                   composer update --prefer-dist --no-interaction --no-ansi --no-progress
                   php vendor/bin/simple-phpunit
 
-    tests-php-8-0-symfony-5-4:
+    tests-php-8-1-symfony-5-4:
```

Comment:
You can remove this test, as the next one test PHP 8.1 and the previous one test Symfony 5.4 (faster test suite and more ecological :) )

Reply:
If you feel strongly about it I will remove it, but I disagree. PHP 8.1 and Symfony 5.4 is a valid supported configuration so I feel we should run the test suite for it

Reply:
I understand your position, but testing all possible configurations is usually a waste of CPU-time in my experience. What we need to test is:

* Support for PHP 8.0 => tests-php-8-0-symfony-5-4
* Support for PHP 8.1 => tests-php-8-1-symfony-6-1
* Support for Symfony 5.4 => tests-php-8-0-symfony-5-4
* Support for Symfony 6.0 => tests-php-8-0-symfony-6-0
* Support for Symfony 6.1 => tests-php-8-1-symfony-6-1

We can safely do this because Symfony test itself against all these PHP versions: we don't need to test that Symfony 5.4 will work with PHP 8.0 *and* PHP 8.1, we can take that for granted.

This is why I think this test-case isn't necessary: it duplicates already tested configurations.

Reply:
Ok, I have removed the two unnecessary test cases